data/mods/Xedra_Evolved/mutations/paraclesians/arvore_mutation_spells.json

```
@@ -413,6 +414,55 @@
     "base_energy_cost": 350,
     "base_casting_time": 400
   },
+  {
+    "id": "arvore_summon_floral_boat",
+    "type": "SPELL",
+    "name": "Viridescent Vessel",
+    "description": "Tree require water but will still drown of submerged.  This spell allows the Arvore to transform a tree into a floral boat, letting them travel safely across water.\n\nYou must be standing <color_yellow>next to a tree</color> and <color_yellow>near the water</color> for the spell to be effective.",
```

```suggestion
    "description": "Tree require water but will still drown if submerged.  This spell allows the Arvore to transform a tree into a floral boat, letting them travel safely across water.\n\nYou must be standing <color_yellow>next to a tree</color> and <color_yellow>near the water</color> for the spell to be effective.",
```
That's what you meant?

---

Yes, thank you.